resources/views/admin/user/index.blade.php

```diff
@@ -0,0 +1,83 @@
+@extends('layouts.admin.base')
+@section('admin_content')
+    @include('layouts.components.notification')
+    <div class="table-agile-info">
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                {{ __('admin.title_list_user') }}
+            </div>
+            <div class="row w3-res-tb">
+                <div class="col-sm-5 m-b-xs">
+                </div>
+                <div class="col-sm-4">
+                </div>
+                <div class="col-sm-3">
+                    <form class="input-group" action="">
+                        <input type="text" class="input-sm form-control" name="key">
+                        <span class="input-group-btn">
+                            <button class="btn btn-sm btn-default" type="submit">Search</button>
+                        </span>
+                    </form>
+                </div>
+            </div>
+            <div class="table-responsive">
+                <table class="table table-striped b-t b-light" id="myTable">
+                    <thead>
+                        <tr>
+                            <th>{{ __('admin.number') }}</th>
+                            <th>{{ __('admin.avatar') }}</th>
+                            <th>{{ __('admin.user_name') }}</th>
+                            <th>{{ __('admin.email') }}</th>
+                            <th>{{ __('admin.phone') }}</th>
+                            <th>{{ __('admin.status') }}</th>
+                            <th style="width:30px;"></th>
+                        </tr>
+                    </thead>
+
+                    <tbody>
+                        @foreach ($Users as $index => $user)
+                            <tr>
+                                <td>{{ ++$index }}</td>
+                                <td><img src="{{ Vite::asset('resources/images/user_avatar.png') }}" height="40" width="40"></td>
+                                <td>
+                                    <h4>{{ $user->user_name }}</h4>
+                                </td>
+                                <td>{{ $user->email }}</td>
+                                <td>{{ $user->phone_number }}</td>
+                                <td>
+                                    <span class="text-ellipsis">
+                                        @if ($user->status === \App\Models\User::ACTIVATED)
```

Comment:
Bị lặp code, sửa:
cách 1: trả về thêm class khi query: approved, not-approved,inactivated
cách 2: viết 1 helper để check class.